mmf/datasets/databases/features_database.py

```diff
@@ -117,5 +117,7 @@ def from_path(self, path):
 
     def _get_feature_path_based_on_image(self, item):
         image_path = self._get_attrs(item)[0]
+        if isinstance(image_path, int):
+            return str(image_path) + ".npy"
```

Comment:
Use format string here:
```suggestion
            return f"{image_path}.npy"
```